iac-extensions/arm/sonarpedia.json

```diff
@@ -0,0 +1,11 @@
+{
+  "rules-metadata-path": "./src/main/resources/org/sonar/l10n/azureresourcemanager/rules/azureresourcemanager",
+  "languages": [
+    "AZURE_RESOURCE_MANAGER"
+  ],
+  "latest-update": "2023-05-08T07:13:45.831563776Z",
```

Comment:
Wondering what is this latest update thing, shouldn't it be today?

Reply:
The rule-api doesn't work with ARM yet. It's a copy from another file.